graylog2-server/src/main/java/org/graylog2/contentpacks/facades/SidecarCollectorConfigurationFacade.java

```diff
@@ -91,11 +91,12 @@ public NativeEntity<Configuration> createNativeEntity(Entity entity,
 
     private NativeEntity<Configuration> decode(EntityV1 entity, Map<String, ValueReference> parameters) {
         final SidecarCollectorConfigurationEntity configurationEntity = objectMapper.convertValue(entity.data(), SidecarCollectorConfigurationEntity.class);
-        final Configuration configuration = Configuration.create(
+        final Configuration configuration = Configuration.createWithoutId(
                 configurationEntity.collectorId().asString(parameters),
                 configurationEntity.title().asString(parameters),
                 configurationEntity.color().asString(parameters),
-                configurationEntity.template().asString(parameters));
+                configurationEntity.template().asString(parameters),
+                Set.of());
```

Comment:
I guess we need to support tags in content packs as well. We can do that in another PR though.

Reply:
I thought about that, but tags might be very specific to your environment, so I think we can live without them..

Reply:
I like your answer 👍 😄